widgets/src/custom-elements/SummaryCardElement.js

```diff
@@ -0,0 +1,41 @@
+import i18next from 'i18next';
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import SummaryCard from 'components/SummaryCard/SummaryCardContainer';
+
+class SummaryCardElement extends HTMLElement {
+  connectedCallback() {
+    const mountPoint = document.createElement('div');
+    this.appendChild(mountPoint);
+
+    const locale = this.getAttribute('locale') || 'en';
+    i18next.changeLanguage(locale);
+
+    const customEventPrefix = 'summarycard.';
+
+    const onError = error => {
```

Comment:
I have added `widgets/src/custom-elements/customEventsUtils.js` for custom event generation, maybe it will prove useful

Reply:
done